lib/ansible/plugins/connection/netconf.py

```diff
@@ -5,68 +5,118 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+
 DOCUMENTATION = """
-    author: Ansible Networking Team
-    connection: netconf
-    short_description: Use netconf to run command on network appliances
+---
+author: Ansible Networking Team
+connection: netconf
+short_description: Provides a persistent connection using the netconf protocol
+description:
+  - This plugin actually forces use of 'local' execution but uses paramiko to
```

Comment:
Do we actually want to talk about `local`. Is that confusing implementation detail?

Who is the audience of this?